mk-files/docs.mk

```
@@ -67,12 +67,14 @@ release-docs-main:
 	$(call dry-run,git push -u origin $(CURRENT_SHORT_MINOR_VERSION)-post) && \
 	if [[ $(CLEAN_VERSION) == *.0 ]]; then \
 		git checkout master && \
+		git checkout -b update-settings-v$(CLEAN_VERSION) && \
```

I've been following a convention of naming the branches after the make target so we can tell where a PR came from more easily
```suggestion
		git checkout -b release-docs-main-v$(CLEAN_VERSION) && \
```